Add TLS configuration options to Chronograf TLS guides (#4065)
influxdata/docs-v2: content/chronograf/v1.8/administration/managing-security.md

**To configure Chronograf to support TLS:**

1. Specify the certificate file using the `TLS_CERTIFICATE` environment variable or the `--cert` CLI option.
2. Specify the key file using the `TLS_PRIVATE_KEY` environment variable or `--key` CLI option.

    {{% note %}}
If both the TLS certificate and key are in the same file, specify them using the `TLS_CERTIFICATE` environment variable (or the `--cert` CLI option).
    {{% /note %}}

> 3. _(Optional)_ Specify which TLS cipher suites to allow using the `TLS_CIPHERS` or `--tls-ciphers` CLI option.

Reviewer: ```suggestion
3. _(Optional)_ To specify which TLS cipher suites to allow, use the `TLS_CIPHERS` environment variable or the `--tls-ciphers` CLI option.
```

    Chronograf supports all cipher suites in the
    [Go `crypto/tls` package](https://golang.org/pkg/crypto/tls/#pkg-constants)
    and, by default, allows all of them.

> 4. _(Optional)_ Specify the minimum and maximum TLS versions to allow with the
>     `TLS_MIN_VERSION` and `TLS_MAX_VERSION` environment variables or the
>     `--tls-min-version` and `--tls-max-version` CLI options.
>     By default, the minimum TLS version allowed is `tls1.2` and the maximum version is
>     unlimited.

Reviewer: ```suggestion
4. _(Optional)_ To specify the minimum and maximum TLS versions to allow, use the
    `TLS_MIN_VERSION` and `TLS_MAX_VERSION` environment variables or the
    `--tls-min-version` and `--tls-max-version` CLI options.
    By default, the minimum TLS version allowed is `tls1.2` and the maximum version is
    unlimited.
```

#### Example with CLI options
```sh
chronograf \
  --cert=my.crt \
  --key=my.key \
  --tls-ciphers=TLS_RSA_WITH_AES_256_CBC_SHA,TLS_AES_128_GCM_SHA256 \
  --tls-min-version=tls1.2 \
  --tls-max-version=tls1.3
```

#### Example with environment variables
```sh
TLS_CERTIFICATE=my.crt \
TLS_PRIVATE_KEY=my.key \
TLS_CIPHERS=TLS_RSA_WITH_AES_256_CBC_SHA,TLS_AES_128_GCM_SHA256 \
TLS_MIN_VERSION=tls1.2 \
TLS_MAX_VERSION=tls1.3 \
chronograf
```

#### Docker example with environment variables
```sh
docker run \
  -v /host/path/to/certs:/certs \
  -e TLS_CERTIFICATE=/certs/my.crt \
  -e TLS_PRIVATE_KEY=/certs/my.key \
  -e TLS_CIPHERS=TLS_RSA_WITH_AES_256_CBC_SHA,TLS_AES_128_GCM_SHA256 \
  -e TLS_MIN_VERSION=tls1.2 \
  -e TLS_MAX_VERSION=tls1.3 \
  chronograf:{{< current-version >}}
```

### Test with self-signed certificates

> In a production environment you should not use self-signed certificates, but for testing it is fast to create your own certificates.

Reviewer: ```suggestion
To test your setup, you can use a self-signed certificate.

{{% warn %}}
Don't use self-signed certificates in production environments.
{{% /warn %}}

```
